dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2415,6 +2921,21 @@
         "value"
       ]
     },
+    "DateTimeSubKind": {
```

Comment:
Is this one needed if `DateTimeResolution` is supposed to be going away?

Reply:
resolutions are still used by custom entity result

Reply:
IIUC, entity resolutions didn't GA in analyze-text, so is there a reason to keep it for custom entities?

Reply:
Custom text is still using the entity definition with resolutions so we cannot remove it without breaking something.

Reply:
When you say "service contract", are you referring to the "service API contract", or is there a different kind of contract in place regarding custom NER?

Reply:
This swagger would not represent what we return from the service.

Reply:
A different team runs custom text, they were not a part of the discussions to move to the new representations of Entity, so to not affect them we created a new entity in parallel to the existing entity.

Reply:
I am afraid this feels like an instance of shipping an organization instead of shipping the product. Is there a reason why they're not included in the discussion? I believe it is best for our customers to have one experience regarding this and providing two not only causes confusion but also unnecessarily increases our public surface and things we need to maintain. /cc @johanste

Reply:
We are removing the resolution objects

Reply:
Thanks! However, I still see some resolutions definitions, e.g. `BaseResolution`.

Reply:
I believe those are for the analyze-conversations apinot the analyze-text api

Reply:
My bad, I opened the wrong commit. However, I think the definition `DateTimeSubKind` being commented on here is still there and should be deleted, no?

Reply:
yes, it is no longer used so I removed it.

Reply:
Awesome, I will mark this thread as resolved then.